Jazz up README & swap out 'terra' for 'Verily Workbench'

emmarogge opened the pull request.

emmarogge committed:
Rename files to use VWB instead of Terra

Rename directory containing First Hour on VWB notebooks, files to use new product name.

emmarogge committed:
Jazz up the README for the First Hour on Verily Workbench directory

emmarogge committed:
Solicit user feedback by providing contact link in README

johntbates commented on first_hour_on_vwb/README.md:
```diff
@@ -0,0 +1,16 @@
+
+## Feedback
+
+If you have feedback about your experience with the First Hour on Verily Workbench materials, we'd love to hear from you. Please reach out to <a href="mailto:emmarogge@google.com?subject=First Hour on VWB Feedback">Emma Rogge</a> to share your thoughts.
```
Suggestion: create a Verily group and add yourself to it.  Then, add that group's email address to this workspace.  This will let you add other people to the support group and potentially even remove yourself, should the need arise.

emmarogge replied:
Great point -- I have now done so.

johntbates reviewed (approved):
Generally LGTM - consider my Verily group suggestion, though.

emmarogge committed:
Direct feedback to new VWB group

Direct feedback to a newly created VWB group, first-hour-on-vwb-feedback@verily-bvdp.com, instead of my LDAP.
